packages/amplify-util-mock/src/CFNParser/stack/index.ts

```diff
@@ -87,22 +87,24 @@ export function getDependencyResources(node: object | any[], params: Record<stri
     return [];
```

Comment:
Only this function `getDependencyResources` is changed. The others are lint changes